src/Stratis.Bitcoin.Features.PoA/CollateralPoAConsensusFactory.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Text;
+using NBitcoin;
+using Stratis.Bitcoin.Utilities.JsonConverters;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class CollateralPoAConsensusFactory : PoAConsensusFactory
+    {
+        public override IFederationMember DeserializeFederationMember(byte[] serializedBytes)
+        {
+            string json = Encoding.ASCII.GetString(serializedBytes);
```

Comment:
What happens if the bytes you pass in here is not a json string?

Reply:
It will return `null`

Reply:
This will be a problem if 51% of fed members umbit invalid data that doesn't translates to json string. So imho no need to fix it, if 51% of fed members are malicious there are way worse things you can do

Reply:
IMO better error handling can be done here (even if 51% is reached you want to throw a proper exception perhaps?)

Reply:
Not urgent though